### PR TITLE
remove seed_ef and use ef for hnsw iterator

### PR DESCRIPTION
**Pull request by zhengbuqian**
kind/improvement

issue: https://github.com/zilliztech/knowhere/issues/500

### CODE REVIEW

**Comment by zhengbuqian** [comment]
kind/improvement

issue: #500

File: src/index/hnsw/hnsw_config.h [hunk 1]
```diff
@@ -45,11 +44,6 @@ class HnswConfig : public BaseConfig {
             .set_range(1, std::numeric_limits<CFG_INT::value_type>::max())
             .for_search()
```
Comment (alwayslove2013) [inline src/index/hnsw/hnsw_config.h:45]: seems that `ef` need for_iterator

**Review by alwayslove2013 — CHANGES_REQUESTED** [review]

File: src/index/hnsw/hnsw_config.h (hunk 1 above)
Comment (zhengbuqian) [reply src/index/hnsw/hnsw_config.h:45]: good catch!

**Comment by zhengbuqian** [comment]
kind/improvement

issue: https://github.com/zilliztech/knowhere/issues/500

**Pull request merged**